1_fetch/src/get_sb_data.R

```diff
@@ -258,11 +258,54 @@ prep_feature_vars <- function(sb_var_data, sites, retain_vars) {
     select(COMID, label, avg_annual) %>%
     pivot_wider(names_from = "label", values_from = "avg_annual")
   
+  #determine dominant physiographic region
+  phys_region <- data %>%
+    select(COMID, contains("_PHYSIO_")) %>%
+    select(-contains("AREA")) %>%
+    group_by(COMID) %>%
+    summarise_all(mean) %>%
+    pivot_longer(!COMID, names_to = "name", values_to = "value") %>%
+    mutate(unit = str_sub(name, 1, 3), 
+           region = str_sub(name, 12))
+  
+  dom_phys_reg <- tibble()
+  for (i in unique(phys_region$COMID)) {
+    phys_reg_comid <- filter(phys_region, COMID == i)
+    
+    dom_phys_reg_comid <- tibble()
+    for (j in unique(phys_reg_comid$unit)) {
+      phys_reg_comid_unit_ties <- filter(phys_reg_comid, unit == j) %>%
+        arrange(desc(value))
+      
+      #if tie for dominant phyiographic region, default to CAT value
+      if (phys_reg_comid_unit_ties[[1,3]] > phys_reg_comid_unit_ties[[2,3]]) {
```

Comment:
Should this line have single brackets `[]` instead of double `[[]]`? All other changes look good to me! I think merging should wait until after #122 is merged so that you can transfer over the _targets data with the script in that PR

Reply:
Either single or double brackets work, I preferred double in this instance because it just returns a numeric value rather than the number with the label. 

I'll review #122 shortly.